httputil/httputil.go

```diff
@@ -0,0 +1,46 @@
+package httputil
+
+import (
+	"net/http"
+	"net/http/httputil"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// PrintRequest ...
+func PrintRequest(request *http.Request) error {
+	if request == nil {
+		return nil
+	}
+
+	dump, err := httputil.DumpRequest(request, true)
```

Comment:
https://golang.org/src/net/http/httputil/dump.go?s=2110:2175#L67 DumpRequestOut seems to be reccomended for outgoing requests.

Reply:
I don't want to change code in this round 🙀

Reply:
Ok